src/models/maze.c

```diff
@@ -79,10 +82,17 @@ void drawCoins(int frameCount)
     }
 }
 
-void drawMaze()
+void drawMazeBlue()
```

Comment:
I'm not sure if it's pertinent to create 2 separate functions for such a little difference.

If we implement the X offset that i've talked about earlier, we can maybe pass an argument to the function to decide if we apply the offset or not to display the white maze. This way, we will always call the same function to draw the maze

Reply:
https://github.com/Livl-Corporation/livl-pacman/pull/73/commits/759412bb9a86dc2b51690348bf24231b3689ae40

Reply:
is it okay now ?